src/FindApprenticeshipTraining/SFA.DAS.FindApprenticeshipTraining.Api/AppStart/AddServiceRegistrationExtension.cs

```diff
@@ -27,6 +27,7 @@ public static void AddServiceRegistration(this IServiceCollection services)
             services.AddTransient<ICachedCoursesService, CachedCoursesService>();
             services.AddTransient<ILocationLookupService, LocationLookupService>();
             services.AddTransient<IRoatpCourseManagementApiClient<RoatpV2ApiConfiguration>, RoatpCourseManagementApiClient>();
+            services.AddTransient<IRoatpV2TrainingProviderService, RoatpV2TrainingProviderService>();
```

Comment:
There is already a client above that points to RoatpV2

Reply:
ok